zanata-war/src/main/webapp/WEB-INF/layout/admin/contact_admin_modal.xhtml

```diff
@@ -32,10 +34,26 @@
       return getContactAdminForm().find('#contact-admin-send-button');
     }
 
+    // Returns a span.
+    function getContactAdminMessageErrorField(){
+      return getContactAdminForm().find('#contactAdminForm\\:messageField\\:input\\:contact-admin-error-message\\:contactAdminForm\\:messageField\\:input\\:contact-admin-message');
```

Comment:
We've tried different solutions for my js training with @carlosmunoz and changed a few things. Here how we went;

We removed the controlling of "send button enability". JSF "required=true" and "max length" validations are enough not to send invalid messages to the admin.

We changed the way of getting error message field from id to class to get rid of ridiculously long id. The function returns a span which cannot be controlled by its visibility. So, again we controlled its text. Carlos also adds that "Both of them can bypassed by someone. Because it is javascript." However, any bypasses cannot send invalid message because jsf validation is doing that on the java side. So no worries.

Finally, I added a label that shows the left characters. It was just an experimental in order to be bias-free to javascript . If you guys are not happy with that, I can remove it. 

Apparently, it works well.